packages/block-editor/src/components/url-input/index.js

```diff
@@ -471,70 +393,59 @@ class URLInput extends Component {
 				role: 'option',
 				tabIndex: '-1',
 				id: `${ suggestionOptionIdPrefix }-${ index }`,
-				ref: this.bindSuggestionNode( index ),
+				ref: bindSuggestionNode( index ),
 				'aria-selected': index === selectedSuggestion,
 			};
 		};
 
-		if (
-			isFunction( renderSuggestions ) &&
-			showSuggestions &&
-			!! suggestions.length
-		) {
-			return renderSuggestions( {
+		if ( isFunction( experimentalRenderSuggestions ) ) {
+			return experimentalRenderSuggestions( {
 				suggestions,
 				selectedSuggestion,
 				suggestionsListProps,
 				buildSuggestionItemProps,
 				isLoading: loading,
-				handleSuggestionClick: this.handleOnClick,
+				handleSuggestionClick: handleOnClick,
 				isInitialSuggestions:
-					__experimentalShowInitialSuggestions &&
-					! ( value && value.length ),
+					showInitialSuggestions && ! ( value && value.length ),
```

Comment:
```suggestion
					showInitialSuggestions && ! value?.length ,
```

Reply:
Updated in f6a9403 (I had forgotten about  ES6 optional chaining, thanks!)